core/src/services/azblob/backend.rs

```diff
@@ -558,9 +558,12 @@ impl Accessor for AzblobBackend {
     }
 
     async fn create_dir(&self, path: &str, _: OpCreateDir) -> Result<RpCreateDir> {
-        let mut req =
-            self.core
-                .azblob_put_blob_request(path, Some(0), None, None, AsyncBody::Empty)?;
+        let mut req = self.core.azblob_put_blob_request(
+            path,
+            Some(0),
+            &Default::default(),
```

Comment:
It's better to use `&OpWrite::default()` so that reader know what happened here.